guides/common/modules/con_best-practices-for-deploying-smartproxies.adoc

```diff
@@ -0,0 +1,18 @@
+[id="best-practices-for-deploying-{smart-proxy-context}_{context}"]
+= Best practices for deploying {SmartProxies}
+
+* Install a {SmartProxyServer} in each of your geographic locations.
+By assigning a {SmartProxy} to each location, you decrease the load on {ProjectServer}, increase redundancy, and reduce bandwidth usage.
++
+[NOTE]
+====
+The maximum number of {SmartProxyServers} that {ProjectServer} can support has no fixed limit.
+It was tested that a {ProjectServer} can support 17 {SmartProxyServers} with 2 vCPUs.
+====
+* Your {ProjectServer} contains an integrated {SmartProxyServer}.
+Do not use the integrated {SmartProxyServer} to communicate with hosts in remote locations because it can lead to suboptimal network utilization.
```

Comment:
```suggestion
Do not use the integrated {SmartProxy} to communicate with hosts in remote locations because it can lead to suboptimal network utilization.
```

I am unsure if this is correct/ if we should make this broad recommendation at this stage. Maybe drop this sentence and instead link to https://docs.theforeman.org/nightly/Tuning_Performance/index-katello.html#Smart_Proxy_Configuration_Tuning_performance-tuning?

Reply:
I think the whole 'best practices' section for deploying capsules might be pretty weak, actually. I merged it with the module on installing capsules. Adding a link to the tuning guide is a very good idea. Thanks!